perror.c

```diff
@@ -0,0 +1,21 @@
+#include "main.h"
+
+/**
+ * _perror - writes a custom error message to stderr
+ * @cmd: erroneous command
+ * Return: void
+ */
+
+void _perror(char *cmd)
+{
+	char *s = "./hsh: 1: ";
```

Comment:
Implement a static integer num instead, to hold the error count as error inputs are entered